src/main/java/servlets/TestNewRecipeServlet.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+package com.google.sps.servlets;
+ 
+import java.util.ArrayList;
+import java.util.List;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+ 
+/** Servlet responsible for creating new comments. */
+@WebServlet("/new-recipe")
+public class TestNewRecipeServlet extends HttpServlet {
+ 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    List<String> searchStrings = new ArrayList<String>();
+    String name = request.getParameter("name-input").toUpperCase();
+    String ingred1 = request.getParameter("ingred-input-1").toUpperCase();
```

Comment:
Are you capitalizing everything so that searching for recipes is case-insensitive?

Reply:
I'm capitalizing everything here for testing, because technically, the search string is in that format (NAME, INGRED1, INGRED2...); this is here so that I can easily fit that format when I store the entry into datastore. But yes, searching for recipes will be "case-insensitive" because of how the search string is formatted.